x/axelarnet/message_handler.go

```diff
@@ -21,17 +22,38 @@ import (
 
 // Fee is used to pay relayer for executing cross chain message
 type Fee struct {
-	Amount    string `json:"amount"`
-	Recipient string `json:"recipient"`
+	Amount          string  `json:"amount"`
```

Comment:
don't we need to know the denomination, too?

Reply:
I feel as weird as you do, but apparently this struct is used only by GMP via IBC and we assume the this denomination as the IBC transfer.

Reply:
IBC only allows one token to be transferred at a time unfortunately. We could add denom for better design and check if it's the same, just adds some more friction for devs, and breaks the API

Reply:
I don't like assuming, especially when the `Fee` struct can be used out of context. Is there any harm in adding the denomination in? Also, why is everything a string again? Maybe add a comment for that?